module/prover/l1/config.go

```diff
@@ -138,9 +145,15 @@ func (prc *ProverConfig) getForkParameters() *lctypes.ForkParameters {
 					Epoch:   prc.MinimalForkSched[Electra],
 					Spec:    &ElectraSpec,
 				},
+				{
+					Version: common.Hex2Bytes("70000038"),
```

Comment:
@yoshidan Where did `"70000038"` come from?
https://github.com/ethereum/consensus-specs/blob/v1.6.0-beta.0/configs/minimal.yaml#L52

Reply:
It is because ethpandaops overrides the fork version.
https://github.com/ethpandaops/ethereum-package/blob/33a0db2f85303add140bbc997bd8ca5d25a46081/src/package_io/constants.star#L127